luigi.cfg

```diff
@@ -0,0 +1,27 @@
+[core]
+    autoload_range=true
+    log_level = INFO
+    local_scheduler = True
+
+[GlobalParams]
+    source=pubmed
+
+[DownloadTask]
+    from_month=2021-12
+    output_dir=luigi-pipeline
+    identifier=
+    ; emtpy string is considered default value
+
+[TopicExtractTask]
+    mesh_topic_db=luigi-pipeline/mesh_topic_db.json
+
+[TopicFilterTask]
+    filter_config=luigi-pipeline/filter-config.jsonl
+
+[ConvertPDFTask]
+    grobid_host=0.0.0.0
+    grobid_port=8070
+
+[AddTask]
+    db_url=luigi-pipeline/my-db.db
+    db_type=sqlite
```

Comment:
```
;Blue Brain Search is a text mining toolbox focused on scientific use cases.
;
;Copyright (C) 2020  Blue Brain Project, EPFL.
;
;This program is free software: you can redistribute it and/or modify
;it under the terms of the GNU Lesser General Public License as published by
;the Free Software Foundation, either version 3 of the License, or
;(at your option) any later version.
;
;This program is distributed in the hope that it will be useful,
;but WITHOUT ANY WARRANTY; without even the implied warranty of
;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
;GNU Lesser General Public License for more details.
;
;You should have received a copy of the GNU Lesser General Public License
;along with this program. If not, see <https://www.gnu.org/licenses/>.

[core]
autoload_range = true
log_level = INFO
local_scheduler = True

[GlobalParams]
source = pubmed

[DownloadTask]
from_month = 2021-12
output_dir = luigi-pipeline
identifier =
; emtpy string is considered default value

[TopicExtractTask]
mesh_topic_db = luigi-pipeline/mesh_topic_db.json

[TopicFilterTask]
filter_config = luigi-pipeline/filter-config.jsonl

[ConvertPDFTask]
grobid_host = 0.0.0.0
grobid_port = 8070

[AddTask]
db_url = luigi-pipeline/my-db.db
db_type = sqlite

```
INI/CFG linting + BBP header.

Reply:
Thank you! Done in [bd77681](https://github.com/BlueBrain/Search/pull/571/commits/bd77681a87f42973760646f19330e053e4b1b712).